src/components/flow/routers/result/helpers.ts

```diff
@@ -110,15 +109,9 @@ export const stateToNode = (
   }
 
   let nodeType = Types.split_by_run_result;
-  let operand = DEFAULT_OPERAND;
+
   const asset = state.result.value;
-  if (asset.type === AssetType.URN) {
-    operand = `@(format_urn(contact.urns.${asset.id}))`;
-  } else if (asset.type === AssetType.Field) {
-    operand = `@contact.fields.${asset.id}`;
-  } else {
-    operand = `@contact.${asset.id}`;
-  }
+  const operand = `@(field(results.${asset.id}, ${state.fieldNumber}, '${state.delimiter}'))`;
```

Comment:
this isn't quite right - string literals in excellent are always with `"` @ericnewcomer 

sorry I should have actually reviewed this!